execution/interface.go

```diff
@@ -21,7 +20,6 @@ type RecordResult struct {
 	Pos       arbutil.MessageIndex
 	BlockHash common.Hash
 	Preimages map[common.Hash][]byte
```

Comment:
we still need batchinfo for recordings. Execution-side doesn't need the ability to read arbitrary batches